tests/parser/EmbeddedPython.cpp

```diff
@@ -71,16 +71,19 @@ BOOST_AUTO_TEST_CASE(INSTANTIATE)
     BOOST_CHECK_MESSAGE(python->enabled(), "Python interpreter must be enabled in a default-constructed Python object when we have Embedded Python support");
     BOOST_CHECK_NO_THROW(python->exec("import sys"));
 
-    Parser parser;
-    Deck deck;
     const std::string python_code = R"(
-print('Parser: {}'.format(context.parser))
-print('Deck: {}'.format(context.deck))
-kw = context.DeckKeyword( context.parser['FIELD'] )
-context.deck.add(kw)
```

Comment:
I guess this tested somewhere else?

Reply:
Yes, in the test below https://github.com/OPM/opm-common/blob/master/tests/parser/EmbeddedPython.cpp#L93